### PR TITLE
[release-4.5] Bug 1889679: Collect installplans

### PR DESCRIPTION
**Pull request by 0sewa0**
A backport (4.5) of a new data enhancement for gathering of installplans.

## Categories

- [ ] Bugfix
- [X] Enhancement
- [X] Backporting
- [ ] Others (CI, Infrastructure, Documentation)

## Sample archive

- `docs/insights-archive-sample/config/instalplans.json`

## Documentation

- `docs/gathered-data.md`

## Unit Tests

- `pkg/gather/clusterconfig/clusterconfig_test.go`

## Privacy

Yes. There are no sensitive data in the newly collected information.

## References

https://issues.redhat.com/browse/CCXDEV-3329
https://issues.redhat.com/browse/INSIGHTOCP-75
https://bugzilla.redhat.com/show_bug.cgi?id=1889679

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,3 +1,9 @@
+GO_PKG=github.com/openshift/insights-operator
+GO111MODULE?=on
+GOFLAGS=-mod=vendor
+export GO111MODULE
+export GOFLAGS
+
```
Comment (tremes) [inline Makefile:6]: I guess this shouldn't be here or am I wrong?

**Review by natiiix — APPROVED** [review]
/lgtm

But it would be great to mention why are we making changes in `config/local.yaml` in an enhancement backport.

**Comment by 0sewa0** [comment]
@natiiix this PR was inherited from Martin back from September so the reason for those changes are unknown to me

**Review by rluders — APPROVED** [review]
/lgtm

**Pull request merged**